### PR TITLE
HW7 gamzinova

### PR DESCRIPTION
**Pull request by gamzinova**
Hi,

Please review my HW7.

Thanks in advance,
Anna

### CODE REVIEW

File: Anna Gamzinova/HW7a_gamzinova.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# arithmetic function
+
+def arithmetic(left_operand, right_operand, operation):
```
Comment (bohdandovbsyh) [inline Anna Gamzinova/HW7a_gamzinova.py]: use annotations when describe parameters

File: Anna Gamzinova/HW7b_gamzinova.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import math
+
+# The function returns perimeter, area and diagonal of the square
+
+
+def square(side_length):
```
Comment (bohdandovbsyh) [inline Anna Gamzinova/HW7b_gamzinova.py]: use annotations when describe function

File: Anna Gamzinova/HW7c_gamzinova.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# is_prime function
+
+def is_prime(number):
```
Comment (bohdandovbsyh) [inline Anna Gamzinova/HW7c_gamzinova.py]: same here

File: Anna Gamzinova/incident_of_the_dog.txt [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+It was nice in the police cell. It was almost a perfect cube, 2 metres long by 2 metres wide by 2 metres high. It contained approximately 8 cubic metres of air. It had a small window with bars and, on the opposite side, a metal door with a long, thin hatch near the floor for sliding trays of food into the cell and a sliding hatch higher up so that policemen could look in and check that prisoners hadn’t escaped or committed suicide. There was also a padded bench.
+
+I wondered how I would escape if I was in a story. It would be difficult because the only things I had were my clothes and my shoes which had no laces in them.
+
+I decided that my best plan would be to wait for a really sunny day and then use my glasses to focus the sunlight on a piece of my clothing and start a fire. I would then make my escape when they saw the smoke and took me out of the cell. And if they didn’t notice I would be able to wee on the clothes and put them out.
+
+I wondered whether Mrs Shears had told the police that I had killed Wellington and whether, when the police found out that she had lied, she would go to prison. Because telling lies about people is called Slander.
```
Comment (bohdandovbsyh) [inline Anna Gamzinova/incident_of_the_dog.txt]: please remove the file from PR.

File: Anna Gamzinova/HW7d_gamzinova.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Removing all the digits from the file with unknown length
+
+import re
+
+
+def no_digits(file):
```
Comment (bohdandovbsyh) [inline Anna Gamzinova/HW7d_gamzinova.py]: use annotations. and better naming is file_path for this parameter

File: Anna Gamzinova/HW7a_gamzinova.py (hunk 1 above)
Comment (gamzinova) [reply Anna Gamzinova/HW7a_gamzinova.py]: @BohdanDovbysh1994 fixed

File: Anna Gamzinova/HW7b_gamzinova.py (hunk 2 above)
Comment (gamzinova) [reply Anna Gamzinova/HW7b_gamzinova.py]: @BohdanDovbysh1994 fixed

File: Anna Gamzinova/HW7c_gamzinova.py (hunk 3 above)
Comment (gamzinova) [reply Anna Gamzinova/HW7c_gamzinova.py]: @BohdanDovbysh1994 done

File: Anna Gamzinova/incident_of_the_dog.txt (hunk 4 above)
Comment (gamzinova) [reply Anna Gamzinova/incident_of_the_dog.txt]: @BohdanDovbysh1994 done

File: Anna Gamzinova/HW7d_gamzinova.py (hunk 5 above)
Comment (gamzinova) [reply Anna Gamzinova/HW7d_gamzinova.py]: @BohdanDovbysh1994 fixed

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**